LibraryComponents/Classes/AndesMessage/View/AndesMessageAbstractView.swift

```diff
@@ -92,7 +92,7 @@ class AndesMessageAbstractView: UIView, AndesMessageView {
 
         if config.isDismissable, let iconName = config.dismissIconName {
             self.dismissButton.isHidden = false
-            self.titleToDismissConstraint.priority = .defaultHigh
+            self.titleToDismissConstraint.priority = .required
```

Comment:
usar .required hace que la app en iOS viejos (no se cuando cambió exactamente el comportamiento) se rompa. Si probas en iOS 10.3 explota:

Porque no podes cambiar de una constraint required a una non required. y tampoco una non required a una required en runtime.


![image](https://user-images.githubusercontent.com/57450033/86294645-59f1de80-bbcb-11ea-8bf4-e0938090c013.png)

Podemos probar usar una que sea 999?

```suggestion
            self.titleToDismissConstraint.priority = .init(rawValue: 999)
```

Reply:
Perfecto, no conocía de ese problema en iOS 10.3. Probé lo que mencionas y anda perfectamente.